.misc/generate_dictionary.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# generate_dictionary.py
+#
+# Generates custom documentation dictionary for SpellCheckBear
+
+import os
+import sys
+from subprocess import call
+from scspell import __file__ as scfile
+
+
+def get_data_dir(prog_name):
+    '''Retrieves a platform-appropriate data directory for the specified
+    program.
+
+    :param str prog_name: The program from which to get the data directory
+    '''
+    if sys.platform == 'win32':
+        parent_dir = os.getenv('APPDATA')
+        prog_dir = prog_name
+    else:
+        parent_dir = os.path.expanduser('~')
+        prog_dir = '.' + prog_name
+    return os.path.normpath(os.path.join(parent_dir, prog_dir))
+
+
+class GenerateDictAction:
+    '''Represents the action of generating a new dictionary based on a
+    custom dictionary.'''
+    lib_dict = os.path.normpath(
+        os.path.join(scfile, '../data/dictionary.txt'))
+    local_dict = os.path.normpath(
+        os.path.join(get_data_dir('scspell'), 'dictionary.txt'))
+    custom_dict = os.path.normpath(
+        os.path.join(os.getcwd(), '.misc/dictionary.txt'))
+
+    mkdir_command = ('mkdir', '-p', '~/.scspell')
```

Comment:
use `os.mkdir` , etc, for file operations.
(This will also fix the CircleCI problem)